pandas/_libs/src/parser/tokenizer.c

```diff
@@ -1433,13 +1433,14 @@ PANDAS_INLINE void uppercase(char *p) {
 int to_boolean(const char *item, uint8_t *val) {
     char *tmp;
     int i, status = 0;
-    int bufsize = sizeof(char) * (strlen(item) + 1);
+    size_t length0 = (strlen(item) + 1);
+    int bufsize = sizeof(char) * length0;
```

Comment:
Can just get rid of `sizeof(char)` here as C standard guarantees this to be 1

Reply:
yeah